apps/admin-gui/src/app/vos/pages/member-detail-page/member-overview/member-overview.component.html

```diff
@@ -68,17 +68,29 @@ <h1 class="page-subtitle">{{'MEMBER_DETAIL.OVERVIEW.RESET_PASSWORD' | translate}
           </div>
         </mat-card-content>
       </mat-card>
-      <div *ngIf="member.sponsored">
-        <mat-card appearance="outlined" class="mat-elevation-z3 mt-4">
-          <mat-card-header>
-            <mat-card-title>
-              <h1 class="page-subtitle">{{'MEMBER_DETAIL.OVERVIEW.SPONSORSHIP' | translate}}</h1>
-            </mat-card-title>
-            <mat-card-subtitle class="mt-n3">
-              {{'MEMBER_DETAIL.OVERVIEW.SPONSORED_BY' | translate}}
-            </mat-card-subtitle>
-          </mat-card-header>
-          <mat-card-content *ngIf="sponsors.length !== 0">
+
+      <mat-card
+        appearance="outlined"
+        class="mat-elevation-z3 mt-4"
+        *ngIf="member.sponsored || enableSponsorButton()">
+        <mat-card-header>
+          <mat-card-title>
+            <h1 class="page-subtitle">{{'MEMBER_DETAIL.OVERVIEW.SPONSORSHIP' | translate}}</h1>
+          </mat-card-title>
+          <mat-card-subtitle class="mt-n3 mb-2" *ngIf="member.sponsored">
+            {{'MEMBER_DETAIL.OVERVIEW.SPONSORED_BY' | translate}}
+          </mat-card-subtitle>
+        </mat-card-header>
+        <mat-card-content>
+          <button
+            *ngIf="enableSponsorButton()"
```

Comment:
Do not call functions from the template, as it is inefficient. Either use a pipe or create a property (something like `sponsorButtonEnabled`) that gets updated whenever sponsors are changed.